include/Evolution/DiffEvol.h

```diff
@@ -52,80 +47,118 @@ struct Constraint {
 };
 
 template <typename T>
-constexpr T mutate(T min, T max, T a, T b, T c) {
-   return std::clamp(a + static_cast<T>(g_mutation * (b - c)), min, max);
+constexpr T mutate(int i, Parameters evo_params, T min, T max, T a, T b, T c) {
+   auto initial = evo_params.mutation_rate_initial;
+   auto final = evo_params.mutation_rate_final;
```

Comment:
`final` to keyword, nie powinien być nazwą zmiennej

Reply:
dopiero przeczytałem, dzięki